code/modules/mob/living/simple_animal/abnormality/aleph/mountain.dm

```diff
@@ -147,7 +187,11 @@
 				StageChange(TRUE)
 
 /mob/living/simple_animal/hostile/abnormality/mountain/PickTarget(list/Targets) // We attack corpses first if there are any
-	if(phase == 1)
+	if(phase == 1 || (phase == 2 && health <= maxHealth*0.5))
+		for (var/mob/living/carbon/human/body in view(20, src))
+			if (body.stat != DEAD || body.health < 0)
```

Comment:
"If body isn't dead or its health is below 0"
What's the intent here..?
Also, code below exists for it already, and as far as I am aware adding things to Targets list won't do anything, you have to return a list.

Reply:
Sniff out dead or dying, honestly this current part is in testing and wasnt the Commit that was Test Merged

Reply:
oh wait yeah this one was intended to give him increased sight of dead and dying people. This DOES work because High Priority and Low Priority checks will be run through this. This is his "in beta" ability to smell out bodies.

Reply:
Also yeah I was very tired writing that check